bin/node/bench/Cargo.toml

```diff
@@ -11,9 +11,9 @@ license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 [dependencies]
 log = "0.4.8"
 node-primitives = { version = "2.0.0", path = "../primitives" }
-node-testing = { version = "3.0.0-dev", path = "../testing" }
-node-runtime = { version = "3.0.0-dev", path = "../runtime" }
-sc-cli = { version = "0.10.0-dev", path = "../../../client/cli" }
+node-testing = { version = "4.0.0-dev", path = "../testing" }
+node-runtime = { version = "4.0.0-dev", path = "../runtime" }
+#sc-cli = { version = "0.10.0-dev", path = "../../../client/cli" }
```

Comment:
I'd prefer that we simply removed these lines instead of commenting them out.

Reply:
absolutely. once I get a green build :-)

Reply:
got a green build so they are gone.